src/components/NavBar.jsx

```diff
@@ -1,45 +1,195 @@
 import React from 'react';
-import { connect } from 'react-redux';
 import styled from 'styled-components';
+import { connect } from 'react-redux';
+import media from '../util/media';
+import Button from './buttons/Button';
 import checkIfAdmin from '../util/admin';
 import OutlinedButton from './buttons/OutlinedButton';
+import AddAgentModal from './modals/add-agent/AddAgentModal';
 import { logout } from '../util/auth';
 import Link from './Link';
 
 const NavigationBar = styled.div`
-  height: 50px;
   padding: 16px;
+  background-color: ${props => props.theme.lightBlack};
   display: grid;
-  grid-template-columns: 1fr 8fr [logout];
+  grid-template-columns: [logo] 200px [content] 1fr;
+  grid-gap: 16px;
   align-items: center;
-  background-color: ${props => props.theme.lightBlack};
+  z-index: 1;
+
+  ${
+    media.phone`
+      grid-template-columns: [main] 1fr [toggle] 24px;
+      grid-template-rows: [logo] 50px [content] 1fr;
+
+      position: ${props => props.collapsed ? 'static' : 'fixed'};
+      top: 0px;
+      left: 0px;
+      right: 0px;
+      bottom: ${props => props.collapsed ? 'static' : '0px'};
+    `
+  }
+`;
+
+const NavBarContent = styled.div`
+  grid-column: content;
+  display: grid;
+  grid-template-columns: [menu] 1fr [new-agent] 250px [logout] 100px;
+  grid-gap: 12px;
+  height: auto;
+
+  ${
+    media.phone`
+      display: ${props => props.collapsed ? 'none' : 'grid'};
+      grid-column: main / -1;
+      grid-row: content;
+      height: 100%;
+
+      grid-template-columns: [main] 1fr;
+      grid-template-rows: [menu] 1fr [new-agent] 50px [logout] 50px;
+      maring-bottom: 32px;
```

Comment:
```suggestion
      margin-bottom: 32px;
```

But if unnoticed then probably redundant.